changelog.d/363.misc

```diff
@@ -0,0 +1 @@
+Remove references to @inlineCallbacks and convert affected functions to async.
```

Comment:
Maybe:

```suggestion
Convert `inlineCallbacks` to async/await.
```

Reply:
I don't think this was updated?

Reply:
See my comment above, nothing has been pushed yet :)